Makefile

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Some people may use the Makefile for running the tests and we have no control about this ...